### ISSUE TITLE
How to generate *.lab file for custom *.wav file

### ISSUE THREAD

**Issue by vinayentc** [BUTSpeechFIT/VBx#4]
Hello,

For the predict.py we need "--in-lab-dir example/vad", how can we generate *.lab file for the custom *.wav file.

Can you please provide more details on this file and its generation/availability.

Thanks.

**Comment by vinayentc** [comment]
Ok, below script I have written to generate *.lab file -

URL - https://github.com/vinayentc/VBx/blob/master/VBx/generate_lab_file.py
Usage - generate_lab_file.py <aggressiveness> <path to wav file>

*.lab file will be generated where the generate_lab_file.py exist.

**Comment by fnlandini** [comment]
Hello,
These recipes make use of the reference (oracle) VAD and for this reason there is not code for producing VAD labels. They can be derived from the rttm files of a dataset and for the case of DIHARD they are even provided in the dataset.
We have shared in the VoxConverse branch code for running an energy-based VAD: https://github.com/BUTSpeechFIT/VBx/tree/v1.1_VoxConverse2020/VAD
That allows for generating the lab files from the wavs. Thank you for sharing your code using webrtcvad. In our experience for the VoxSRC challenge (working with VoxConverse), when tuning the threshold of the energy VAD we obtained better results than with webrtc. The energy VAD can be applied on other datasets but we do not include it in the master branch.

I will close the issue as the master recipe does not consider any system VAD for the moment.
